### PR TITLE
Fixed small spelling + punctuation errors

### PR DESCRIPTION
**Pull request by ghost**
:cl: 
spellcheck: Fixes small typographical errors on flight suits and implanters.
/:cl:

### CODE REVIEW

**Comment by bgobandit** [comment]
line 338 of flightsuit.dm: should be "received"

i before e except after c

File: code/modules/clothing/spacesuits/flightsuit.dm [hunk 1]
```diff
@@ -335,7 +335,7 @@
 		if(brake || stabilizer)
 			brake = FALSE
 			stabilizer = FALSE
-			usermessage("Warning: Sensor data is not being recieved from flight shoes. Stabilizers and airbrake modules OFFLINE!", 2)
+			usermessage("Warning: Sensor data is not being received from flight shoes. Stabilizers and airbrake modules OFFLINE!", 2)
```
Comment (Cyberboss) [inline code/modules/clothing/spacesuits/flightsuit.dm]: recieved

**Review by Cyberboss — APPROVED** [review]

**Comment by nicbn** [comment]
http://www.gingersoftware.com/english-online/spelling-book/misspelling/receive-recieve-receiv

**Comment by optimumtact** [comment]
Did you know there are more exceptions to that rule than things that follow it?

**Comment by bgobandit** [comment]
or if sounding like A as in neighbor and weigh
or if it's fucking weird, as in fucking weird

**Comment by ClosingBracket** [comment]
the world moves in mysterious ways

**Pull request merged**